### DISCUSSION TITLE
Automatic translation reminders

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#23] [Team Posts]
Hello,

since the app **enroute** is now being translated to more languages, I set up a script that regularly checks for untranslated strings and reports them to the relevant team members. 

**Explanation:** Notification is done by means of "GitHub Issues". If you have a look at the GitHub page of the project "enrouteText", you will see a number of issues, with the name "Unfinished Translations - XX", where XX is the language code. The script will automatically open/close/reopen those issues. I have assigned the issues to the relevant team members, so GitHub will inform you automatically if something with the issue changes.

I am still in "experimentation phase". There are a few pending string changes in the main app, so please expect a fair number of notifications in the next few days. Once the dust settles, the number of notifications should go down drastically. Please let me know if this system works for you! Also, please let me know if you have any concerns here.

@woodwind-enr I am going through your translations. I will come back to you once I am done. Till then, please leave the translation file to me.

Best wishes,

Stefan.

**Comment by Dzordzik** [comment]
Hi Stephan,

if you remembe, I was try to make translation and I finish it, but problem 
is I was doing it in downloaded file from Github, instead of push-pul scheme
you told me so I spend time on it but without efect, which is my problem, 
but I´m not experienced in how to do it correctly by git. If I will able to 
do it by some easier way for me, no problem to do it. Between this time I 
was made very comprehensive CZ translation for another navi app (3-5x more 
texts than in your app) and this translation is finished. It wasn´t problem 
for me, because it was made by translation of data file in some application 
in style entext>cztext, so very easy and fast.




I don´t know how exactly do it by git function and your email with 
explanation don´t help me to know it.




Jirka




---------- Původní e-mail ----------
Od: Stefan Kebekus <notifications@github.com>
Komu: Akaflieg-Freiburg/enroute-translation <enroute translation@noreply.
github.com>
Datum: 3. 7. 2020 10:21:50
Předmět: [Akaflieg-Freiburg/enroute-translation] Automatic translation 
reminders (#1) 
"


Hello,

since the app enroute is now being translated to more languages, I set up a 
script that regularly checks for untranslated strings and reports them to 
the relevant team members.

Explanation: Notification is done by means of "GitHub Issues". If you have a
look at the GitHub page of the project "enrouteText", you will see a number 
of issues, with the name "Unfinished Translations - XX", where XX is the 
language code. The script will automatically open/close/reopen those issues.
I have assigned the issues to the relevant team members, so GitHub will 
inform you automatically if something with the issue changes.

I am still in "experimentation phase". There are a few pending string 
changes in the main app, so please expect a fair number of notifications in 
the next few days. Once the dust settles, the number of notifications should
go down drastically. Please let me know if this system works for you! Also, 
please let me know if you have any concerns here.

@woodwind-enr(https://github.com/woodwind-enr) I am going through your 
translations. I will come back to you once I am done. Till then, please 
leave the translation file to me.

Best wishes,

Stefan.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
(https://github.com/orgs/Akaflieg-Freiburg/teams/enroute-translation/discussions/1)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/AKU4DSAYNVSUK6DEBX5U3Q3RZWIJHANCNFSM4OPS65XA)
. 
"

**Comment by acrovato** [comment]
Hi Stefan,

Good idea to setup automated reminders for translations! Everything is working fine on my side.  
I will update the French translations regularly on my branch and PR from time to time (or before release, at your best convenience).

**Comment by kebekus** [comment]
@acrovato Thanks. I have one request, though. You have write access to enrouteText. Please write there directly and do not use pull-requests for your translations. I trust that you get the translations right. I do not know much French and would like to reduce the load on my overflowing inbox.

**Comment by acrovato** [comment]
@kebekus No worries, will do!
